subprojects/docs/src/docs/release/notes.md

```diff
@@ -37,6 +37,23 @@ Switch your build to use Gradle @version@ by updating your wrapper:
 
 `./gradlew wrapper --gradle-version=@version@`
 
+## Transforming dependency artifacts on resolution
+
+A dependency’s artifacts can take many forms, some of which will be published but others may not be.
```

Comment:
This sentence is confusing. Why don't we just talk about a simple Java module that gets published in the usual (unobfuscated) way, and that you want all your dependencies (including this one) obfuscated, so you employ an artifact transform to do it.

Reply:
Can you make a suggestion? I don't find this confusing, since for me, that is what artifact transforms are about: convert one form of an artifact to another one since it has not been published in the desired form.

Reply:
```suggestion
A dependency’s artifacts can take many forms, and sometimes you might need them in a form that is not readily available in a published form.
```